src/light-box/share.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+import { Share } from 'react-native';
+
+import { BRAND_COLOR } from '../styles';
+
+export default (url: string) => {
+  const shareOptions = {
+    message: url,
+    title: 'Shared using Zulip',
```

Comment:
👍
Here I need the translation as a string so probably I'd use [injection api](https://github.com/yahoo/react-intl/wiki/API#injection-api). Maybe we could do the translation outside this function? Did a quick test with this [component](https://gist.github.com/Sam1301/8acd225190d327de9fd5e18c452ea168#file-lightboxcontainer-js-L81).